pro/src/screens/User/UserProfile.tsx

```diff
@@ -1,7 +1,28 @@
+import { ProfileForm } from 'new_components/ProfileForm'
 import React from 'react'
+import { UserIdentityResponseModel } from 'apiClient/v1'
+import { userProfileForms } from './UserProfileForms'
 
-const ProfileScreen = (): JSX.Element => {
-  return <h1>Profil</h1>
+const ProfileScreen = ({
+  ...identityData
+}: UserIdentityResponseModel): JSX.Element => {
```

Comment:
Le point des adapter et de ne pas utiliser les type de l'api directement dans nos screen en ajoutant une couche de serialisation "type de api -> type de l'application" 

```suggestion
export interface IProfileScreenProps {
  firstName: string
  lastName: string
}

const ProfileScreen = ({
  firstName,
  lastName,
}: IProfileScreenProps): JSX.Element => {
```